draft-ietf-quic-tls.md

```diff
@@ -213,6 +209,11 @@ own key update mechanism {{key-update}}.
 ~~~
 {: #tls-full title="TLS Handshake with 0-RTT"}
 
+{{tls-full}} omits the EndOfEarlyData message, which is not used in QUIC; see
+{{remove-eoed}}. Likewise, neither ChangeCipherSpec nor KeyUpdate messages are
+used by QUIC. ChangeCipherSpec is redundant in TLS 1.3; see {{compat-mode}}.
+QUIC has its own key update mechanism; see {{key-update}}.
```

Comment:
Worth noting that it's not possible to send CCS anywya?

Reply:
Not here, but I'll do that below.